requirements.yml

```diff
@@ -1,5 +1,7 @@
 ---
 roles:
-  - geerlingguy.postgresql
+  - name: geerlingguy.postgresql
+    src: https://github.com/fao89/ansible-role-postgresql.git
```

Comment:
This is not ideal, but I am OK with doing it for the current versions of the installer until geerlingguy merges our PR.

Reply:
I agree this is not ideal, actually, I did this to facilitate my local tests, and I did so many rebases that I forgot.
I can move it to CI, as before